src/qt/forms/optionsdialog.ui

```diff
@@ -238,7 +238,7 @@
              <string>Whether to show coin control features or not.</string>
             </property>
             <property name="text">
-             <string>Enable coin &amp;control features</string>
+             <string>En&amp;able coin control features</string>
```

Comment:
you've introduced a new ambiguity within this file, now this option collides with the shortcut to switch to the `Window` tab within the `Options Window`. If you test this new shortcut, the `Enable Coin Control features` shortcut will not actuate.

Reply:
Thanks, Jarol, for catching this! 

I have reverted to the original change done for this option in this PR.
Addressed in 5fde8fb.